Automattic/wp-calypso
# Checkout: Update Thank You page copy for non-domain purchases

This pull request addresses part of #2484 by updating the feature sections of the `Thank You` page with the new copy for chargeback, site redirect, Google Apps, and Jetpack plans purchases:

![screenshot](https://cloud.githubusercontent.com/assets/594356/13393088/892c4938-dedf-11e5-8fa7-658064a0067c.png)

#### Testing instructions
 
1. Run `git checkout update/thank-you-copy` and start your server
2. Purchase a [site redirect](http://calypso.localhost:3000/domains/add/site-redirect/YOURSITE) and check the `Thank You` page
3. Purchase [Google Apps](http://calypso.localhost:3000/domains/add/site-redirect/YOURSITE) for an existing domain registration and check the `Thank You` page
4. Purchase a chargeback and check the `Thank You` page
5. Switch to a Jetpack site
6. Purchase a [Jetpack plan](http://calypso.localhost:3000/plans/YOURSITE) and check the `Thank You` page
 
#### Additional notes

Jetpack plans cannot be tested until https://github.com/Automattic/wp-calypso/pull/3617 is deployed. Icons are addressed in https://github.com/Automattic/wp-calypso/pull/3578.

#### Reviews
